custom/qgroundcontrol.qrc

```diff
@@ -290,8 +290,17 @@
         <file alias="QGroundControl/Controls/CameraRecordButton.qml">../src/QmlControls/CameraRecordButton.qml</file>
         <file alias="QGroundControl/Controls/CameraGalleryButton.qml">../src/QmlControls/CameraGalleryButton.qml</file>
         <file alias="QGroundControl/Controls/CommonTopMenu.qml">../src/QmlControls/CommonTopMenu.qml</file>
-        <file>MissionHistoryItem.qml</file>
+        <file alias="QGroundControl/Controls/MissionHistoryItem.qml">../src/QmlControls/MissionHistoryItem.qml</file>
```

Comment:
Why do you add this item in this  file?

Reply:
![image](https://github.com/argosdyne/AELS_QGC/assets/92770122/f8e9cb6c-ec46-4359-8289-1ced06762e26)

This error kept coming up, so I changed the path to be the same as the other file.